modules/op-release-notes-1-8.adoc

```diff
@@ -609,4 +609,16 @@ config   1.8.1     True
 // https://issues.redhat.com/browse/SRVKP-2422
 
 * Before this update, cluster administrators could not provide a `config.json` file to the `Buildah` cluster task for accessing a container registry. With this update, cluster administrators can provide the `Buildah` cluster task with a `config.json` file by using the `dockerconfig` workspace.
-// https://issues.redhat.com/browse/SRVKP-2424
\ No newline at end of file
+// https://issues.redhat.com/browse/SRVKP-2424
+
+[id="release-notes-1-8-2_{context}"]
+== Release notes for {pipelines-title} General Availability 1.8.2
+
+With this update, {pipelines-title} General Availability (GA) 1.8.2 is available on {product-title} 4.10 and 4.11.
+
+[id="fixed-issues-1-8-2_{context}"]
+=== Fixed issues
+
+* Before this update, the `git-clone` task failed when cloning a repository using SSH keys. With this update, the role of the non-root user in the `git-init` task is removed, and the SSH program looks in the `$HOME/.ssh/` for the correct keys.
```

Comment:
s/in the `$HOME/.ssh/` for the correct keys. /in the `$HOME/.ssh/` directory/folder for the correct keys.